src/server/storage/websocket.js

```diff
@@ -894,6 +903,11 @@ function WebSocket(storage, mainLogger, gmeConfig, gmeAuth, workerManager) {
                     })
                     .then(function (newToken) {
                         data.webgmeToken = newToken;
+                        //TODO this should probably come from authenticator and not the request!!!
+                        if (gmeConfig.authentication.enable === true &&
+                            gmeConfig.authentication.azureActiveDirectory.enable === true) {
+                            data.aadToken = cookies[gmeConfig.authentication.azureActiveDirectory.cookieId] || null;
+                        }
```

Comment:
I realize the webgmeToken was already in the `data` but we need to make sure we don't debug-log that incoming data - at least not without sanitizing.

Reply:
I'll check if anyone logs the data, but these tokens are kinda public info, and we are talking about server side logging...